### PR TITLE
ENH/DOC: Add missing parameter descriptions and rename `chunk_size`

### LINKED ISSUE #85: Missing parameter descriptions in classify-sklearn
There's no documentation for the following parameters:

```
  --i-reads PATH                  Artifact: FeatureData[Sequence]  [required]
  --i-classifier PATH             Artifact: TaxonomicClassifier  [required]
  --p-chunk-size INTEGER          [default: 262144]
  --p-n-jobs INTEGER              [default: 1]
  --p-pre-dispatch TEXT           [default: 2*n_jobs]
```

### PR DESCRIPTION
**Pull request by BenKaehler**
Added some input and parameter descriptions for `classify-sklearn`.

Fixes #85

### CODE REVIEW

File: q2_feature_classifier/classifier.py [hunk 1]
```diff
@@ -182,7 +186,15 @@ def classify_sklearn(reads: DNAIterator, classifier: Pipeline,
...
-                            'confidence estimates for the first 100 reads.'
+                            'confidence estimates for the first 100 reads.',
+        'chunk_size': 'Number of reads to process in each batch.',
+        'n_jobs': 'The maximum number of concurrently worker processes. If -1 '
+                  'all CPUs are used. If 1 is given, no parallel computing '
+                  'code is used at all, which is useful for debugging. For '
+                  'n_jobs below -1, (n_cpus + 1 + n_jobs) are used. Thus for '
+                  'n_jobs = -2, all CPUs but one are used.',
+        'pre_dispatch': '"all" or expression, as in "3*n_jobs". The number of '
+                        'batches (of tasks) to be pre-dispatched.'
```
Comment (ElDeveloper) [inline q2_feature_classifier/classifier.py]: What does it mean for a job to be pre-dispatched?

File: q2_feature_classifier/classifier.py [hunk 2]
```diff
@@ -182,7 +186,15 @@ def classify_sklearn(reads: DNAIterator, classifier: Pipeline,
                             'complement will cause reads to be reversed '
                             'and complemented prior to classification. '
                             'Default is to autodetect based on the '
-                            'confidence estimates for the first 100 reads.'
+                            'confidence estimates for the first 100 reads.',
+        'chunk_size': 'Number of reads to process in each batch.',
```
Comment (ElDeveloper) [inline q2_feature_classifier/classifier.py]: A bit out of the scope of this PR, but would it make sense to rename this parameter as `number_of_reads` or `reads`.

Comment (BenKaehler) [reply q2_feature_classifier/classifier.py]: I would be fine with this so long as it made it clear that this parameter was related to chunking the input. Something like `reads_per_batch` or `reads_per_chunk`.

File: q2_feature_classifier/classifier.py [hunk 3]
```diff
@@ -182,7 +186,15 @@ def classify_sklearn(reads: DNAIterator, classifier: Pipeline,
...
+        'n_jobs': 'The maximum number of concurrently worker processes. If -1 '
+                  'all CPUs are used. If 1 is given, no parallel computing '
+                  'code is used at all, which is useful for debugging. For '
+                  'n_jobs below -1, (n_cpus + 1 + n_jobs) are used. Thus for '
+                  'n_jobs = -2, all CPUs but one are used.',
+        'pre_dispatch': '"all" or expression, as in "3*n_jobs". The number of '
+                        'jobs (of tasks) to be dispatched during execution.'
```
[the remark this answers is not in the document]
Comment (thermokarst) [reply q2_feature_classifier/classifier.py]: Oh good catch - I was trying to model the phrasing after the sklearn docs, but I was accidentally reading the wrong docstring! Yikes! I am good with what you had originally, with or without changing to `reads_per_batch` or `reads_per_chunk`. Sorry for causing confusion there!

**Comment by thermokarst** [comment]
@BenKaehler: I [reverted](https://github.com/qiime2/q2-feature-classifier/pull/89/commits/2232ee63a653ac1cd92795d09c16ca424f0de7ce) my crazy commit. Sorry!

**Comment by thermokarst** [comment]
I would prefer to see that as a separate PR, personally --- this one is nice and concise right now, unless you think that this PR doesn't make sense without the changes necessary for #90 (or maybe for some other reason). Tomorrow is our PR submission freeze for the 2017.9 cycle, so it would be pretty sweet to see these descriptions make it in!

**Comment by BenKaehler** [comment]
Ok, I've changed `chunk_size` to `reads_per_batch`. Merge when ready!

**Comment by ElDeveloper** [comment]
Thanks everyone!

**Pull request merged**